bot/exts/evergreen/emoji_count.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+import logging
+import random
+from typing import Dict, Optional
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours, ERROR_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class EmojiCount(commands.Cog):
+    """Command that give random emoji based on category."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    def embed_builder(self, emoji: dict) -> discord.Embed:
+        """Generates an embed with the emoji names and count."""
+        embed = discord.Embed()
+        embed.color = Colours.orange
+        embed.title = "Emoji Count"
+        embed.timestamp = datetime.datetime.utcnow()
+        if len(emoji) == 1:
+            for key, value in emoji.items():
+                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
+                embed.set_thumbnail(url=random.choice(value).url)
+        else:
+            msg = ''
+            for key, value in emoji.items():
+                emoji_choice = random.choice(value)
+                emoji_info = f'There are **{len(value)}** emojis in the **{key}** category\n'
+                msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}'
+            embed.description = msg
+        log.trace('Emoji embed built')
+        return embed
+
+    @staticmethod
+    def generate_invalid_embed(ctx: commands.Context) -> discord.Embed:
+        """Genrates error embed."""
+        embed = discord.Embed()
+        embed.color = Colours.soft_red
+        embed.title = random.choice(ERROR_REPLIES)
```

Comment:
```suggestion
        embed = discord.Embed(
            color=Colours.soft_red,
            title=random.choice(ERROR_REPLIES)
        )
```

Reply:
is it make much difference?

Reply:
This is current style. Other cogs use same style for embed building.